odoo/osv/expression.py

```diff
@@ -923,7 +926,7 @@ def parent_of_domain(left, ids, left_model, parent=None, prefix=''):
                     records = records[parent_name] - records.browse(parent_ids)
                 domain = [('id', 'in', list(parent_ids))]
```

Comment:
```suggestion
                parent_ids = list(parent_ids)
                domain = [('id', 'in', parent_ids)]
```
same trick as above 🤔

Reply:
True, we can avoid an iteration. Thanks for pointing this out :)